docs/data/date-pickers/date-range-picker/ResponsiveDateRangePickers.js

```diff
@@ -6,26 +6,40 @@ import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { Unstable_NextDateRangePicker as NextDateRangePicker } from '@mui/x-date-pickers-pro/NextDateRangePicker';
 import { Unstable_MobileNextDateRangePicker as MobileNextDateRangePicker } from '@mui/x-date-pickers-pro/MobileNextDateRangePicker';
 import { Unstable_DesktopNextDateRangePicker as DesktopNextDateRangePicker } from '@mui/x-date-pickers-pro/DesktopNextDateRangePicker';
+import { Unstable_StaticNextDateRangePicker as StaticNextDateRangePicker } from '@mui/x-date-pickers-pro/StaticNextDateRangePicker';
+import { pickersLayoutClasses } from '@mui/x-date-pickers/PickersLayout';
 
-export default function ResponsiveDateRangePicker() {
+export default function ResponsiveDateRangePickers() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <DemoContainer>
-        <DemoItem label="For mobile">
-          <MobileNextDateRangePicker
+        <DemoItem label="Desktop variant">
+          <DesktopNextDateRangePicker
             defaultValue={[dayjs('2022-04-07'), dayjs('2022-04-10')]}
           />
         </DemoItem>
-        <DemoItem label="For desktop">
-          <DesktopNextDateRangePicker
+        <DemoItem label="Mobile variant">
+          <MobileNextDateRangePicker
             defaultValue={[dayjs('2022-04-07'), dayjs('2022-04-10')]}
           />
         </DemoItem>
-        <DemoItem label="Responsive">
+        <DemoItem label="Responsive variant">
           <NextDateRangePicker
             defaultValue={[dayjs('2022-04-07'), dayjs('2022-04-10')]}
           />
         </DemoItem>
+        <DemoItem label="Static variant">
+          <StaticNextDateRangePicker
+            defaultValue={[dayjs('2022-04-07'), dayjs('2022-04-10')]}
+            sx={{
+              backgroundColor: 'red',
+              [`& > .${pickersLayoutClasses.contentWrapper}`]: {
+                alignItems: 'center',
+                backgroundColor: 'green',
+              },
```

Comment:
What are those background colors here for? 🤔 
```suggestion
              [`& > .${pickersLayoutClasses.root}`]: {
                alignSelf: 'center',
              },
```

Reply:
Oh I did not expect to push that :laughing: 
Working on the alignment topic made me realize that the static pickers are passing `sx` and `className` to the view instead of the layout root, which is not what we want.
I am trying to fix it.

Reply:
Would it be OK for you to fix this aligment in a follow up ?
My experiments are changing way to many things to go on a doc PR

Reply:
Sure. it's not a big deal by any stretch of imagination. 🙈 👍